nep-X1.mediawiki

```diff
@@ -27,17 +27,19 @@ Neo N3 has the native <code>ContractManagement</code> contract that is responsib
 {
   "name": "destroy",
   "safe": false,
-  "parameters": [],
+  "parameters": not restricted,
```

Comment:
This is not JSON. And it's incorrect if want a standard everyone knows how to implement and use. We're fine with `[]`. People with `destroy(some, things, that, are, contract, specific)` are just doing what they want and not this NEP.